modules/acquia_cms_image/acquia_cms_image.install

```diff
@@ -19,6 +19,12 @@ function acquia_cms_image_install($is_syncing) {
   if (!$is_syncing) {
     _acquia_cms_common_editor_config_rewrite(TRUE);
 
+    // If installed from profile then return.
+    global $install_state;
+    if (isset($install_state['active_task']) && $install_state['active_task'] == "install_profile_modules") {
+      return;
+    }
+
```

Comment:
Also verify that it works on site install with existing configurations.

Reply:
Yes, it works with:
- `./vendor/bin/drush si minimal && ./vendor/bin/drush  in acquia_cms_image -y && ./vendor/bin/drush cex`
- `./vendor/bin/drush si minimal --existing-config`